.env.development_local

```diff
@@ -1,12 +1,5 @@
+# NOTE: Variables need to be prefixed using REACT_APP_:
+# https://create-react-app.dev/docs/adding-custom-environment-variables/
+
 # GATEWAY
 REACT_APP_GATEWAY=http://localhost:8000/api/v1/
-PORT=3000
-
-# To login using keycloak:
-REACT_APP_KEYCLOAK_REALM=datapunt-ad-acc
-REACT_APP_AUTH_URL=https://iam.amsterdam.nl/auth/
-REACT_APP_CLIENT_ID=wonen-top-frontend
-
-
-REACT_APP_BASEPATH=/
-REACT_APP_SENTRY_ENV=development
```

Comment:
Will these propagate from `.env.development` or do we need to leave them in? Seems to work without them.

Reply:
It will only propagate if you use the `.local` suffix (e.g. `.env.development.local`). This filename will not get any variables propagated from `.env.development`.